R/__ddl_by_AC.R

```diff
@@ -0,0 +1,230 @@
+#' Create input module.
+#'
+#' Template for creating a simple module that will put up input widgets and produce `tdata`.
+#'
+#' Define any inputs necessary to connect to a remote data source and produce data,
+#' as well as a function (`on_submit`) that will create the desired data sets.
+#'
+#' One of the inputs must be an action button (or action link) called "submit".
+#' When clicked, the `on_submit` function will be run.
+#'
+#' `on_submit` must take one argument called `inputs`,
+#' which will be a list of all input elements defined in the UI function except `"submit"`.
+#' The function body must contain all code necessary to obtain the desired data sets and nothing else.
+#' Do not include a return statement, simply assign your data sets to appropriate variables.
+#'
+#' Clicking "submit" will run the function provided in `on_submit`.
+#' The obtained data sets will be packed into a `tdata` object.
+#' The body of `on_submit` will be recorded in the resulting `tdata`.
+#'
+#' The `mask` argument can be used to mask input values used as arguments in the recorded code.
```

Comment:
```suggestion
#' The `mask` argument can be used to mask input values used as arguments in the recorded code. So that sensitive parts of code (like password or username) are substituted with `mask`ed values.
```